easybuild/easyconfigs/e/ELPA/ELPA-2023.05.001_fix_AVX512_support.patch

```diff
@@ -0,0 +1,107 @@
+# What: Fixes support for AVX512 on AMD CPUs
```

Comment:
Can you elaborate? Is there an upstream issue? What is the error you see when you don't have this? (and is that at build or runtime?)

Reply:
done